docs/docs/aztec/developer/noir-contracts/getting-started.md

```diff
@@ -1,26 +0,0 @@
-# Getting started
-
-Please consider using the [TUTORIAL-TEMPLATE](../../TUTORIAL-TEMPLATE.md) for standalone guides / tutorials.
-
-## Installing Noir
-
-To write a Noir Contract, you need to write Noir, and to write Noir, you need to [install Nargo](https://noir-lang.org/getting_started/nargo_installation).
-
-## Installing Noir tooling
-
-There are a number of tools to make writing Noir Contracts more pleasant. See [here](https://github.com/noir-lang/awesome-noir#get-coding).
-
-## Quick start
-
-Download an Aztec Box. (Josh / Ze to build :) ).
-
-Or, if you don't want to do that, here's more detail on doing it all yourself:
-
-:::danger TODO
-TODO
-:::
-
-## Creating a new contract package
-
-See [here](https://github.com/AztecProtocol/aztec-packages/tree/master/yarn-project/noir-contracts#creating-a-new-contract-package)
-
```

Comment:
Any of this copyable to the new 'getting started' section?

Reply:
Sure. Right now it is in the `contracts/main` so when you click on contract. But think the general getting started is to be overhauled. Some content were a little split among different files.

Reply:
Ah, cool, thanks!